website/sidebars.json

```diff
@@ -8,8 +8,9 @@
       "research",
       "community",
       "contributing",
-      "contributors",	
+      "contributors",
       "glossary",
+      "ens",
```

Comment:
I'm wondering if this page would be a better fit inside the build section?

Reply:
I don't think so, this is for users primarily and just a convenience thing at that. I thought it fits well besides the Ledger instructions, but I'll defer to team consensus. Given how much wallet content we have (wallet list, ledger, and now this) it might even make sense to make a wallet sub section or some kind, but we can discuss that later.

Reply:
Yeah I agree -- having a wallet sub-category underneath one of the sections would keep the content a bit more organized 👍